bindings/java/src/main/native/Makefile.am

```diff
@@ -4,8 +4,9 @@
 #
 
 topdir=$(abs_top_builddir)
+java_build_dir=./build-java
```

Comment:
can use `$(builddir)/` instead of `./`

Reply:
![image](https://user-images.githubusercontent.com/1121987/60433145-c038cf80-9c0c-11e9-94cd-6c1aff063a38.png)

Reply:
Done